### ISSUE TITLE
Exclusive Dragon Heads?

### ISSUE THREAD

**Issue by Tomanex** [DragonMountsTeam/DragonMounts2-Expanded#33]
Skeleton, Wither, Zombie, Ender, Aether and Nether drop their heads under certain conditions, unless all Dragons drop their heads under certain conditions.
Of course only one variant instead of two.
![ZFTUR4_3UW9VFSKO](https://github.com/user-attachments/assets/b4165b55-ac82-4759-adca-6c893faa281f)

**Comment by 2190303755** [comment]
Do you mean to add dragon heads to 1.12.2?

**Comment by Tomanex** [comment]
> Do you mean to add dragon heads to 1.12.2?

Yes. I don't know if it's hard or unnecessary but I do have ideas on what to do with them.
